discovery_engine/lib/src/domain/document_manager.dart

```diff
@@ -155,13 +155,25 @@ class DocumentManager {
   /// Fails if [sec] is negative or [id] does not identify an active document.
   Future<void> addActiveDocumentTime(
     DocumentId id,
-    DocumentViewMode mode,
+    DocumentViewMode viewMode,
     int sec,
   ) async {
     if (sec < 0) {
       throw RangeError.range(sec, 0, null);
     }
 
+    if (cfgFeatureStorage) {
+      return _engine.timeSpent(
+        TimeSpent(
+          id: id,
+          smbertEmbedding: Embedding.fromList([]),
+          viewTime: Duration(seconds: sec),
+          viewMode: viewMode,
+          reaction: UserReaction.neutral,
```

Comment:
:art: maybe comment on the choice of values
```suggestion
          smbertEmbedding: Embedding.fromList([]), // unused
          viewTime: Duration(seconds: sec),
          viewMode: viewMode,
          reaction: UserReaction.neutral, // unused
```